Lib/asyncio/selector_events.py

```diff
@@ -895,7 +895,10 @@ def __init__(self, loop, sock, protocol, waiter=None,
         self._eof = False
         self._paused = False
         self._empty_waiter = None
-
+        if hasattr(socket.socket, 'sendmsg'):
```

Comment:
The Python decides if `sendmsg()` is present on compile time.
I suggest using the global boolean flag here, as we do already for `_HAS_IPv6`.

Reply:
Sure.